src/main/java/com/exasol/adapter/dynamodb/documentfetcher/dynamodb/GenericTableAccessParameters.java

```diff
@@ -0,0 +1,89 @@
+package com.exasol.adapter.dynamodb.documentfetcher.dynamodb;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.exasol.adapter.dynamodb.documentnode.DocumentValue;
+import com.exasol.adapter.dynamodb.documentnode.dynamodb.DynamodbNodeToAttributeValueConverter;
+import com.exasol.adapter.dynamodb.documentnode.dynamodb.DynamodbNodeVisitor;
+
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+/**
+ * This class groups the common parameters of Scan and Query document fetchers.
+ */
+public class GenericTableAccessParameters implements Serializable {
+    private static final long serialVersionUID = -822082800908345226L;
+    private final String tableName;
+    private final Map<String, String> expressionAttributeNames;
+    private final Map<String, DocumentValue<DynamodbNodeVisitor>> expressionAttributeValues;
+    private final String filterExpression;
+    private final String projectionExpression;
+
+    /**
+     * Create an instance of {@link GenericTableAccessParameters}.
+     *
+     * @param tableName                 name of the table to scan
+     * @param expressionAttributeNames  map of placeholders for attribute names
+     * @param expressionAttributeValues map of placeholders for attribute values
+     * @param filterExpression          filter expression
+     * @param projectionExpression      projection expression
+     */
+    GenericTableAccessParameters(final String tableName, final Map<String, String> expressionAttributeNames,
```

Comment:
I think you can use setters or a builder for this class too.

Reply:
It is only created by `DynamodbScanDocumentFetcher.Builder` and `DynamodbQueryDocumentFetcher.Builder`
Since that I thought it makes no sense to create a builder that is used by a builder.